src/main/java/edu/ucla/library/libcal/handlers/ProxyHandler.java

```diff
@@ -86,14 +91,20 @@ public ProxyHandler(final Vertx aVertx, final JsonObject aConfig) {
     public void handle(final RoutingContext aContext) {
         final HttpServerResponse response = aContext.response();
         final String path = aContext.request().path();
+        final String method = aContext.request().method().name();
         final String originalClientIP = aContext.request().remoteAddress().hostAddress();
         final Cidr4Trie<String> allowedIPs = buildAllowedNetwork(myConfig.getString(Config.ALLOWED_IPS).split(COMMA));
 
         if (isOnNetwork(new Ip4(originalClientIP), allowedIPs)) {
             final String receivedQuery = path.concat(
                     aContext.request().query() != null ? QUESTION_MARK.concat(aContext.request().query()) : EMPTY);
             myTokenProxy.getBearerToken().compose(token -> {
-                return myApiProxy.getLibCalOutput(token, receivedQuery).map(myMapper::decode);
+                if (GET.equals(method)) {
+                    return myApiProxy.getLibCalOutput(token, receivedQuery).map(myMapper::decode);
+                } else {
```

Comment:
If I'm reading the router correctly, we pass all other requests here to this handler. What if someone tries to submit a non-GET / non-POST request? Should we handle GET/POST and then return a crafted error for any other method type?

Reply:
IMO it's best for us to simply pass everything through to LibCal; as I've suggested [earlier](https://github.com/UCLALibrary/libcal-proxy/pull/7#discussion_r897039329), I don't think this app should be concerned with validating requests.

So instead of branching here based on the HTTP method, LibCalProxyService could have a single method that has an "HTTP method" parameter, which it could pass to [`WebClient#requestAbs`](https://vertx.io/docs/apidocs/io/vertx/ext/web/client/WebClient.html#requestAbs-io.vertx.core.http.HttpMethod-java.lang.String-).

Reply:
Switched to doing all methods, simply accepting what HTTP method comes in from client.